### DISCUSSION TITLE
how to create 'open with atom' item in context menu ubuntu right click

### DISCUSSION THREAD

**Discussion by moein9gh** [atom/atom#22794] [General]
how to create 'open with atom' item in context menu ubuntu right click

**Comment by icecream17** [comment]
Maybe this screenshot can help someone answer
![image](https://user-images.githubusercontent.com/58114641/127745533-0f7ece8f-9b65-4bc6-b1fb-cebb0ccd28ef.png)

**Reply by moein9gh** [reply]
thanks guy
